source/designers-guide/smarty-plugins/index.md

```diff
@@ -216,3 +216,7 @@ The plugin uses widgets to embed templates. Widgets are self-contained parts of
 In this case, an HTTP request is triggered internally within the system, resulting in widgets being completely dynamic elements which are not cached.
 
 For further information read this blog: <a href="{{ site.url }}/blog/2016/07/11/on-action-tags/">On action tags</a>
+
+
+## Custom Smarty plugins
+To register custom smarty plugins please see [register custom smarty plugins](https://developers.shopware.com/designers-guide/smarty/#register-custom-smarty-plugins)
```

Comment:
you could use `{{ site.url }}/designers-guide/....` here

Reply:
`{{ site.url }}` is only possible in a-tag. But I changed it.